IsraelHiking.Common/Extensions/GeoJsonExtensions.cs

```diff
@@ -170,16 +170,21 @@ public static string GetTitle(this IFeature feature, string language)
 
         public static string GetDescription(this IFeature feature, string language)
         {
-            if (feature.Attributes.Exists(FeatureAttributes.DESCRIPTION + ":" + language))
+            string[] suffixes = {":" + language, ""};
```

Comment:
Changing this method might have other side effects. Did you check where it is being used and that it's safe to change it?

Reply:
I did not do a through check.

Reply:
I did a brief check and I don't think this function should be change, I think there should be a different one or a parameter that defaults to previous usage.